chef-server-compliance-delivery-devops/azuredeploy.json

```diff
@@ -0,0 +1,240 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "location": {
+      "defaultValue": "westus",
+      "allowedValues": [
+        "westus",
+        "eastus",
+        "centralus",
+        "eastus2"
+      ],
+      "type": "string",
+      "metadata": {
+        "description": "Location where all resources are deployed"
+      }
+    },
+    "vm-adminName": {
+      "defaultValue": "",
+      "type": "string",
+      "metadata": {
+        "description": "Chef Complaince Virtual Machine Administrator Name"
+      }
+    },
+    "vm-adminPassword": {
+      "defaultValue": "",
+      "type": "securestring",
+      "metadata": {
+        "description": "Chef Complaince Virtual Machine Administrator Password"
+      }
+    },
+    "vm-size": {
+      "defaultValue": "Standard-D2",
+      "type": "string",
+      "allowedValues": [
+        "Standard_D1",
+        "Standard_D2",
+        "Standard_D3"
+      ],
+      "metadata": {
+        "description": "Chef Compliance Virtual Machine Size"
+      }
+    },
+    "dnsLabelPrefix": {
+      "type": "string",
+      "metadata": {
+        "description": "Chef Compliance DNS prefix"
+      }
+    },
+    "baseUrl": {
+      "type": "string",
+      "defaultValue": "https://raw.githubusercontent.com/uday-sg/azure-quickstart-templates/add-chef-p2p/chef-server-compliance-delivery-devops",
```

Comment:
Is this

https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/chef-server-compliance-delivery-devops